tools/cldr-code/src/main/java/org/unicode/cldr/util/Rational.java

```diff
@@ -273,6 +273,15 @@ public Rational pow(int i) {
         return new Rational(numerator.pow(i), denominator.pow(i));
     }
 
+    public Rational pow(Rational exponent) {
+        if (exponent.equals(Rational.ONE)) {
+            return this;
+        }
+        // Temporary quick implementation
+        double resultDouble = Math.pow(this.doubleValue(), exponent.doubleValue());
```

Comment:
For any fractional exponents (like Beaufort, both to and from) we can't remain in Rational space. We might be able to get slightly better (if we want to).

Add a Rational pow(int exponent), which creates a rational from numerator.pow(exponent) / denominator.pow(exponent).

Add a Rational root(int integer), that takes the nth root. That's where (other than for 1) we have to get into BigDecimal space.

The Rational pow(Rational exponent) becomes this.pow(exponent.numerator).root(exponent.denominator)

[I think this works; need to double-check]

BTW, if we ever did log units, we'd have to add another field... https://en.wikipedia.org/wiki/Logarithmic_scale#Logarithmic_units

Reply:
I can look at improving this in the second PR for this ticket.